scripts/generate.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if ! command -v gomplate &>/dev/null; then
+  echo "gomplate not found in PATH"
```

Comment:
Instead of asking users to install this, should we build an image like https://github.com/open-policy-agent/gatekeeper/blob/master/Makefile#L404
Then we can use it as part of `make generate`. WDYT?

Reply:
Updated the PR.